web/app/frontend/src/components/screen_containers/FormsScreenContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import { FormDefinition } from "core-js-api-client/lib/types/types";
+import React from "react";
+
+import { FormsScreenContainerProps } from "../../types/screens";
+import client from "../../utils/clients";
+import { FormsScreen } from "../screens/FormsScreen";
+
+export const FormsScreenContainer = ({
+    navigation,
+    route,
+}: FormsScreenContainerProps) => {
+    const [forms, setForms] = React.useState<FormDefinition[]>();
+    const [isLoading, setIsLoading] = React.useState(true);
+    React.useEffect(() => {
+        client().listForms({}).then((data) => {
+            setIsLoading(false);
+            setForms(data.response?.items);
+        });
+    });
+    return (
+        <FormsScreen
```

Comment:
you could also show FormsScreen only when isLoading. E.g. return a loading view when isLoading is true, otherwise return FormsScreen. Way you could be sure that forms is going to be there in the FormsScreen component. Not super necessary though.